### PR TITLE
Add prac_09 files

### PR DESCRIPTION
**Pull request by Quinn368**
@Ajeetha0701
@ChristineYamin
@ivy-naw-mth

### CODE REVIEW

File: prac_09/taxi_simulator.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    bill_to_date += cost
+    print(f"Bill to date: ${bill_to_date:.2f}")
+    return bill_to_date
+
+
+def display_taxis(taxis):
+    """Display the list of taxis."""
+    for i, taxi in enumerate(taxis):
+        print(f"{i} - {taxi}")
+
```
Comment (ChristineYamin) [inline prac_09/taxi_simulator.py:67]: Nice coding

File: prac_09/band.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        return f"{self.name} ({musician_info})"
+
+    def add(self, musician):
+        """Add a musician to the band."""
+        self.musicians.append(musician)
+
+    def play(self):
+        """Simulate the band playing by calling the play method of each musician."""
+        for musician in self.musicians:
+            print(musician.play())
```
Comment (ChristineYamin) [inline prac_09/band.py:20]: I think your code is right

File: prac_09/taxi_simulator.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+MENU = "q)uit, c)hoose taxi, d)rive"
+
+
+def main():
+    """Program of taxi simulator with choosing a taxi and its distance, showing the bill."""
+    print("Let's drive!")
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    current_taxi = None
+    bill_to_date = 0
```
Comment (ivy-naw-mth) [inline prac_09/taxi_simulator.py:12]: Good code, I think I should also use "bill_to_date"! But for now, I use a different name. ^^

File: prac_09/taxi_simulator.py (hunk 1 above)
Comment (Ajeetha0701) [reply prac_09/taxi_simulator.py:67]: great coding

File: prac_09/band.py (hunk 2 above)
Comment (Ajeetha0701) [reply prac_09/band.py:20]: your code looks good

File: prac_09/taxi_simulator.py (hunk 3 above)
Comment (Ajeetha0701) [reply prac_09/taxi_simulator.py:12]: your code looks good

**Pull request merged**